ansible/roles/test/files/acstests/acltb_test.py

```diff
@@ -239,6 +239,38 @@ def runAclTests(self, dst_ip, dst_ip_blocked, src_port, dst_ports):
         tests_passed += (1 if res else 0)
         print "Test #11 %s" % ("PASSED" if res else "FAILED")
 
+	#Creates a ICMP packet
+	pkt0 = simple_icmp_packet(
+                                eth_dst = self.router_mac,
+                                eth_src = self.dataplane.get_mac(0, 0),
+                                ip_src = "10.0.0.1",
+                                ip_dst = dst_ip,
+                                icmp_type=8,
+                                icmp_code=0,
+                                ip_ttl = 64
+                            )
+        #exp_pkt = pkt.deepcopy()
+        exp_pkt0 = simple_icmp_packet(
+                                eth_dst = self.dataplane.get_mac(0, 0),
+                                eth_src = self.router_mac,
+                                ip_src = "10.0.0.1",
+                                ip_dst = dst_ip,
+                                icmp_type=8,
+                                icmp_code=0,
+                                ip_ttl = 63
+                            )
+							
+        # Test #12 - Verify IP protocol & source IP match
+        pkt = pkt0.copy()
+        exp_pkt = exp_pkt0.copy()
+        pkt['IP'].src = "10.0.0.2"
+        exp_pkt['IP'].src = "10.0.0.2"
+        pkt['IP'].proto=0x1
```

Comment:
Just FYI, proto is already 1, since you created the packet using `simple_icmp_packet`.